xversionmessage.h

```diff
@@ -0,0 +1,125 @@
+// Copyright (C) 2018 The Bitcoin Unlimited developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_XVERSIONMESSAGE_H
+#define BITCOIN_XVERSIONMESSAGE_H
+
+
+#include "protocol.h"
+#include "serialize.h"
+// #include "sync.h"
+// #include "tinyformat.h"
+// #include "utilstrencodings.h"
```

Comment:
to be removed? 

wonder if we could mving all the code inside `src/seeder` in the main BU BCH client repo, so to avoid to copy over here and maintain in sync `xversion*` files

edit: I see you proposed the same in the PR description

Reply:
yes.  Doing it would be a large job compared to this PR though, but I think would be easier than cleaning up this entire code base.  So we should consider doing it if we get time to do a cleanup.